Integrations/MongoDB/MongoDB.py

```diff
@@ -0,0 +1,312 @@
+from CommonServerPython import *
+
+from json import JSONDecodeError
+from typing import List, Union, Tuple
+
+from bson.objectid import ObjectId
+from pymongo import MongoClient
+from pymongo.collection import Collection
+from pymongo.database import Database
+from pymongo.errors import OperationFailure
+from pymongo.results import InsertManyResult, UpdateResult, DeleteResult
+
+CONTEXT_KEY = "MongoDB(val._id === obj._id)"
+
+
+class Client:
+    def __init__(
+        self,
+        urls: List[str],
+        username: str,
+        password: str,
+        database: str,
+        ssl: bool = True,
+    ):
+        self._client = MongoClient(urls, username=username, password=password, ssl=ssl)
+        self.db: Database = self._client.get_database(database)
+
+    def is_collection_in_db(self, collection: str) -> bool:
+        return collection in self.db.list_collection_names()
+
+    def get_collection(self, collection: str) -> Collection:
+        if self.is_collection_in_db(collection):
+            return self.db.get_collection(collection)
+        return_error(
+            f"MongoDB: Collection '{collection}' has not found in database '{self.db.name}'"
+        )
+
+    def get_entry_by_id(self, collection: str, alert_ids: List[str]) -> List[dict]:
+        collection_obj = self.get_collection(collection)
+        entries: List[dict] = list()
+        for alert_id in alert_ids:
+            results = collection_obj.find({"_id": ObjectId(alert_id)})
```

Comment:
why not use `find_one()` ?

Reply:
Because this command allows entering multiple ids.

Reply:
so the description is misleading

Reply:
> looks good
> 
> 1. convert to pack
> 2. `"` -> `'`
> 3. no need for proxy param?
> 4. you seem to be populating not too much to context data (for example just IDs), populate more
> 5. use `return_error` only in the `except` clause. in the code raise exceptions
> 6. make sure you have docstrings and type hints to all functions
> 7. no unit test ???



> looks good
> 
> 1. convert to pack
> 2. `"` -> `'`
> 3. no need for proxy param?
> 4. you seem to be populating not too much to context data (for example just IDs), populate more
> 5. use `return_error` only in the `except` clause. in the code raise exceptions
> 6. make sure you have docstrings and type hints to all functions
> 7. no unit test ???

I prefer using `"` and stayed consistent (as we have not standard with it).
 There's no such thing as a proxy in MongoDB.
The data populated depends on the object returned. I can only assure `_id` will get back.
All of the missing docstring functions are just straight forward.
There' are not function here required unittest.